frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -277,199 +274,192 @@ export function FeatureFlag(): JSX.Element {
                         </Col>
                     </Row>
 
-                    {enabledFeatureFlags[FEATURE_FLAGS.MULTIVARIATE_SUPPORT] && (
```

Comment:
Only change from here to the bottom of the file was removing this check and indentation automatically adjusted.